src/appEvent.js

```diff
@@ -1,6 +0,0 @@
-const { webContents } = require("electron")
-module.exports = (event, data) => {
-    webContents.getAllWebContents().forEach(content => {
-        content.send(event, data)
-    })
-}
```

Comment:
This file was moved, but git didn't recognise it as a file move. So that's why it shows up as deleted over here, and will show up as added somewhere else.